card/mod/edit/set/all/form.rb

```diff
@@ -177,7 +180,7 @@ def explicit_form_prefix
   end
 
   def form_context
-    (form_root? || !form_root) ? self : parent
+    form_root? || !form_root ? self : parent
```

Comment:
parens around last three would be clearer

Reply:
parens around last three would change the logic. rubocop's auto correction removed those parens